dataactvalidator/app.py

```diff
@@ -39,22 +42,39 @@ def before_request():
     def handle_response_exception(error):
```

Comment:
Agreed that we probably want to combine this with the generic Exception handler somehow. Maybe just pull out the shared logic into a helper function?